LibCV/PrePARE/PrePARE.py

```diff
@@ -535,7 +535,7 @@ def ControlVocab(self, ncfile, variable=None, print_all=True):
         # -------------------------------------------------------------------
         varid = cmip6_cv.setup_variable(variable_cmor_entry,
                                         self.dictVar['units'],
-                                        self.dictVar['_FillValue'][0],
```

Comment:
is this due to the switch from cdms2 to netcdf?

Reply:
cdms2 was returning values as numpy ndarrays even if they were just one value.  netCDF4 returns numpy scalars for single values.